### ISSUE TITLE
Fly.io is hiring!

### ISSUE THREAD

**Issue by noodlknits** [TandoorRecipes/recipes#2222]
Hello lovely people!
I am a support engineer at Fly.io and we are hiring for multiple positions, you can browse through them and read the descriptions [here](https://fly.io/jobs/)! We hire a bit differently, so don't worry about education, work experience, or a resume. Did you teach yourself to code as a 15yo hacker? Apply! Did you not go to college, but feel like you could do one of the jobs? Apply! Trust me, I was not "qualified" for my current position and likely would not have gotten it at another company, but we love to hire unorthodox candidates and we love to give people a chance who wouldn't normally get one. We are aware that not all talent comes from education and experience in a tech position, and we love and want to nurture that.

Do you want to know why you should work for us from my standpoint? Great! (idc if you do, I'm sharing anyways)
### Reasons I personally love to work at Fly.io:

- My coworkers are all really lovely, intelligent, and genuine people. I love working with them and I would genuinely want to be friends with them personally. They are funny and kind and always willing to help.
- Fly.io really cares for their employees. I know all companies will say this, but this is the first time I have ever _felt_ it. They do a lot to show they view their employees as real people with lives and not just a means to success. Our time outside of work is valued and we are encouraged to live it.
- I actually love the work I do! This is the first time I have been able to confidently say "I am good at my job!" It never really feels like work and that makes it feel like cheating. 
- I have been able to grow here and take on new responsibilities based on my interests and skills, adding further to the feeling that I belong and am good at what I am doing.

In conclusion, it is my very humble opinion that you should apply if there is a position you're interested in (and check back if there's not! We have more coming down the line!). In fact it is my haughty opinion that you should apply! 

Also, in case you think this is like solicitation or idk whatever, there was an alignment chart of where to share job listings and Github issues was "Chaotic Neutral," and I very much took this alignment chart as a challenge, but really you should apply and everything I've said here is absolutely true, please feel free to reach out to me if you have any questions!
